nginx.conf

```diff
@@ -37,9 +37,9 @@ http {
 
   map_hash_max_size 64;
   map_hash_bucket_size 256;
-  map $host $sts {
+  map $name $sts {
     default "max-age=31536000; preload";
-    {{env "CLOUD_GOV_HOST"}} "max-age=31536000; includeSubDomains; preload";
+    ~^{{env "INCLUDE_SUBDOMAINS"}}$ "max-age=31536000; includeSubDomains; preload";
```

Comment:
I think you want parentheses around the subdomains.

Reply:
I don't think it matters, but I will add them.